seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -2636,7 +2636,7 @@
         "<blockquote>function plusThree(num) {<br>  return num + 3;<br>}<br>var answer = plusThree(5); // 8</blockquote>",
         "<code>plusThree</code> takes an <dfn>argument</dfn> for <code>num</code> and returns a value equal to <code>num + 3</code>.",
         "<hr>",
-        "Create a function <code>timesFive</code> that accepts one argument, multiplies it by <code>5</code>, and returns the new value."
+        "Create a function <code>timesFive</code> that accepts one argument, multiplies it by <code>5</code>, and returns the new value. You can call the <code>timesFive</code> function to test it."
```

Comment:
"call" might still be a confusing term at this point. I think it would be easier if we put a comment on the line in the code that says:
```
// modify the number on this line to test your function
```